launch/demo_rviz.launch.py

```diff
@@ -40,7 +40,9 @@ def load_file(package_name, file_path):
     try:
         with open(absolute_file_path, "r") as file:
             return file.read()
-    except (EnvironmentError):  # parent of IOError, OSError *and* WindowsError where available
+    except (
+        EnvironmentError
+    ):  # parent of IOError, OSError *and* WindowsError where available
```

Comment:
Better remove the parentheses:
```suggestion
    except EnvironmentError:  # parent of IOError, OSError *and* WindowsError where available
```